scripts/create_project_table.py

```diff
@@ -0,0 +1,137 @@
+import os
+import glob
+import yaml
+from typing import List
+
+FOLDER_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+OUTPUT_PATH = os.path.join(os.path.dirname(__file__), "../README.md")
```

Comment:
Better: use `pathlib`, and reuse the root directory.
Also "OUTPUT" is incorrect (we also read this file), and "FOLDER" is a little vague.

```suggestion
from pathlib import Path
from typing import List
import yaml


ROOT_PATH = pathlib.Path(__file__).parent
README_PATH = ROOT_PATH / "README.md"
```

Reply:
I don’t understand—where else should type hints be added?